src/pages/seulgi/Login/Login.js

```diff
@@ -1,7 +1,62 @@
 import React from 'react';
+import { useNavigate, Link } from 'react-router-dom';
+import { useState } from 'react';
+import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  const navigate = useNavigate();
+  const [id, setId] = useState('');
+  const [pw, setPw] = useState('');
+
+  const goToMain = () => {
+    navigate('/main-seulgi');
+  };
+
+  const handleIdInput = event => {
+    setId(event.target.value);
+  };
+
+  const handlePwInput = event => {
+    setPw(event.target.value);
+  };
```

Comment:
```suggestion
 const [input, setInput] = useState({ id:"", password: ""})

const handleInput = e => {
    const { name, value } = e.target;
    setInput({
        ...input,
        [name] : value,
        })
     }
```

이렇게 handleInput 함수로 한번에 통합 가능합니다!